src/api/.rubocop_todo.yml

```diff
@@ -674,13 +674,14 @@ Rails/RakeEnvironment:
     - 'lib/tasks/dev/sre.rake'
     - 'lib/tasks/migrate_options_yml.rake'
 
-# Offense count: 4
+# Offense count: 5
 # Configuration parameters: ForbiddenMethods, AllowedMethods.
 # ForbiddenMethods: decrement!, decrement_counter, increment!, increment_counter, insert, insert!, insert_all, insert_all!, toggle!, touch, touch_all, update_all, update_attribute, update_column, update_columns, update_counters, upsert, upsert_all
 Rails/SkipsModelValidations:
   Exclude:
     - 'app/jobs/update_released_binaries_job.rb'
     - 'db/data/20210505160000_backfill_add_kind_to_path_element.rb'
+    - 'db/data/20250312082028_set_architectures_available_field_to_false.rb'
```

Comment:
This, I don't get it. This PR introduces a data migration, and you include it into the RuboCop TODO file... Does it mean we should address this new offense in the TODO file later on?

Reply:
No, we won't. It's in the todo file because rubocop complaints about using `update_all` and thus bypassing the validations, but we are ok in not doing so here.

Maybe it's better to just write a comment in the data migration and remove it from the rubocop todo file if we aren't going to fix it...

Reply:
What about now?

Reply:
Now it looks good to me. :+1: